content-services/7.2/develop/repo-ext-points/content-transformers-renditions.md

```diff
@@ -218,8 +218,7 @@ changed by resetting the following Alfresco global property.
 local.transform.pipeline.config.dir=shared/classes/alfresco/extension/transform/pipelines
 ```
 
-Local transform-core-aio docker image might need a [newer version of image and approach](#adding-pipelines-and-failover-transforms-to-a-t-engine)
-with providing config directly to the T-Engine.
+Direct override is only possible in 3.0.0 version of alfresco-transform-core and higher. To override local transform-core-aio T-Engine configuration, the docker image needs a newer version of the image and follow this approach providing configuration directly to the T-Engine: [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine).
```

Comment:
@Amy-Hyland  I think this makes more sense. What do you think?
Direct override is only possible in the 3.0.0 version of alfresco-transform-core and above. To override the local transform-core-aio T-Engine configuration, the docker image must use a newer version of the image. Follow this approach and provide the configuration directly to the T-Engine: [Adding pipelines and failover transforms to a T-engine](#adding-pipelines-and-failover-transforms-to-a-t-engine).

Reply:
@mstrankowski Does Oliver's suggestion still convey the technical details correctly? Can I incorporate his changes?